app/services/cloud_vision_service.rb

```diff
@@ -1,16 +1,13 @@
 class CloudVisionService
   class << self
     def artwork(value)
-      response = conn(value).post("/v1/images:annotate")
+      response = Faraday.post("https://vision.googleapis.com/v1/images:annotate?key=#{ENV['key']}", "{'requests':[{'image':{'source':{'imageUri':\"#{value}\"}},'features':{'maxResults':5,'type':'LABEL_DETECTION'}}]}",{'Content-Type': 'application/json'})
```

Comment:
can we update this method be updated with:
Faraday.post(url) do |req|
   req.params[:key] = ENV['key']
   req.params['reqests'] =
and so on